_site/index.html

```diff
@@ -346,6 +346,37 @@ <h3>T-Z</h3>
             </p>
           </div>
         </section>
+        <section class="block__word list">
+          <p class="word__signal word__signal__tool">
+            Tool of Oppression
+          </p>
+          <p class="word__title" id="ableism">
+            ableism
+            <span class="word__speech">
+              noun
+            </span>
+          </p>
+          <blockquote class="word__definition">
+            A system that places value on people’s bodies and minds based on societally constructed ideas of normalcy, intelligence and excellence. These constructed ideas of normalcy, intelligence and excellence are deeply rooted in anti-Blackness, eugenics and capitalism.
+          </blockquote>
+          <cite>— Talia "TL" Lewis, a working definition</cite>
+          <div class="block__type">
+            <p class="word__type">
+              Impact
+            </p>
+            <p class="word__breakdown">This form of systemic oppression leads to people and society determining who is valuable or worthy based on people’s appearance and/or their ability to satisfactorily produce, excel and “behave.” Importantly, you do not have to be disabled to experience ableism.</p>
```

Comment:
I have a pet peeve of starting sentences with "This." Instead, I'd suggest:

> As a form of oppression, ableism leads to people and society determining..."

And, I'd like to append to the end "Disabled people can also reinforce ableism internally (manifesting as harm to themselves) and externally (harm to other disabled people).